Arrays/README.md

```diff
@@ -1 +1,36 @@
+# Array
 
```

Comment:
- An array is a collection of similar type items stored at contiguous memory locations.  This makes it easier to calculate the position of each element by simply adding an offset to a base value, i.e., the memory location of the first element of the array. The base value is index 0 and the difference between the two indexes is the offset. 
- In python array is implemented using list data structure.